src/Tribe/Editor/Attendee_Registration.php

```diff
@@ -117,6 +117,10 @@ public function render() {
 		.accordion-header.tribe_attendee_meta {
 			display:none;
 		}
+
+		.accordion-content.tribe_attendee_meta {
+			display: block;
```

Comment:
Um, why not remove the accordion, or at least just remove the `display:none` style above (L 117-119) instead of overriding it?

This is the issue - it's in an accordion and it shouldn't be anymore. Accordions hide their sections by default. This should really be a template change in the end - should have been when it got moved.

Reply:
If we _have_ to reuse the same template, we should add the opening class (or remove the inline style?) so it's not collapsed.

Reply:
Nope, I misread that. So the line I put is `accordion-content`, the line above is `accordion-header`. The call to render the accordion doesn't allow for a flag that sets the accordion as open or closed.

Reply:
I agree on the template change though. I'd love to fix it, but will need a pass to make sure it doesn't affect other parts of the plugin and other plugins.

Reply:
So the accordion is still used in the classic editor. I pulled out the meta content into its own admin view and rendered that instead.

Reply:
Nice!